tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/CombinationListingFeatureContext.php

```diff
@@ -168,7 +171,12 @@ private function assertListedCombinationsProperties(array $expectedDataRows, arr
             Assert::assertSame(
                 $expectedCombination['combination name'],
                 $editableCombinationForListing->getCombinationName(),
-                'Unexpected combination'
+                'Unexpected combination name'
+            );
+            Assert::assertSame(
+                $expectedCombination['combination reference'],
```

Comment:
I believe not all tests have been adapted because there is an error for missing index `combination reference` Either all tests need to be updated, or you can just "protect" this test inside an `isset` block